libsddc/libsddc.cpp

```diff
@@ -186,6 +197,31 @@ int sddc_set_rf_mode(sddc_t *t, enum RFMode rf_mode)
     return 0;
 }
 
+int sddc_support_pga(sddc_t *t)
+{
+    switch(t->handler->getModel())
+    {
+        case RadioModel::RX888r2:
+        case RadioModel::RX999:
```

Comment:
I believe RX888 also supports pga

Reply:
I copied this from here https://github.com/ik1xpv/ExtIO_sddc/blob/master/ExtIO_sddc/tdialog.cpp#L75